src/PhpEvaluator/FilePhpEvaluator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Dhii\Output\PhpEvaluator;
+
+use Exception;
+
+/**
+ * A PHP evaluator that directly evaluates PHP files via `include`.
```

Comment:
Actually it looks like you are using `require`

Reply:
Yea, but for PHP `require` and `include` are the same thing, except for what happens if the file isn't there. Still, maybe better use `include` in the actual code. What do you think?

Reply:
I'm biased towards `require` as I'd prefer my applications to stop execution if they encounter an error, rather than continue "partially" running. They either jog till the very end or not at all. No limping allowed.

But that's just my opinion. I can understand if others disagree since we're dealing with output here, which tends to not be mission critical.

Reply:
Yes, I understand, and I feel the same way. One of my points was that `include` and `require` share the same mechanic. But yea.

Reply:
Yeah. But the class doc still needs to match what is actually used.

Reply:
Fixed. Now using `include`.